scheduler/src/cook/config.clj

```diff
@@ -234,7 +234,7 @@
                                      {:json-value "config-impersonation"})))
      :rate-limit (fnk [[:config {rate-limit nil}]]
                    (let [{:keys [auth-bypass-limit-per-m expire-minutes job-launch job-submission user-limit-per-m]
-                          :or {auth-bypass-limit-per-m 10000
+                          :or {auth-bypass-limit-per-m 400
```

Comment:
600 to be consistent with user-limit-per-m.

Reply:
Done